### DISCUSSION TITLE
Any way to disable recommendations and comments?

### DISCUSSION THREAD

**Discussion by arjpar** [arichornlover/uYouEnhanced#111] [Q&A]
I think on YouTube Lite Plus you have the ability to disable recommendations and the comments section. I was wondering if this is possible in uYouEnhanced? I don't think so.

**Comment by arichornlover** [comment]
I can port the options from YTLite to uYouEnhanced and I will even credit the creator of the code.

**Reply by arjpar** [reply]
That would be amazing. Thank you!

**Reply by arichornlover** [reply]
Hey I was able to add an option called [Hide all videos under player](https://github.com/arichornlover/uYouEnhanced/commit/3bca764334a2f857d77cb85d232d00a59dd63347) but as for the comment section, I’ll see if I can find a way to implement that.

**Reply by arjpar** [reply]
Hey @arichornlover thanks for the latest update - it's great. Is it possible to have an option to disable the home feed/tab?
